include/El/core/Matrix/impl.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include "impl_cpu.hpp"
 
-#ifdef HYDROGEN_HAVE_CUDA
+#ifdef HYDROGEN_HAVE_GPU
```

Comment:
No corresponding comment on the `#endif`. 😮

Reply:
I wasn't doing it religiously for 3-line blocks. Do you think I should? I'm happy to do so (well, perl is happy to do so).

Reply:
🤷 When I was reviewing, all the previous 3-line blocks I had seen had it, but some of the subsequent ones don't, and I didn't mark them all. Maybe not worth it.

Reply:
Will cover all these blocks in a future PR.